plugins/scaffolder/src/components/fields/RepoUrlPicker/AzureRepoPicker.tsx

```diff
@@ -50,13 +84,30 @@ export const AzureRepoPicker = (props: {
         required
         error={rawErrors?.length > 0 && !owner}
       >
-        <InputLabel htmlFor="ownerInput">Owner</InputLabel>
-        <Input
-          id="ownerInput"
-          onChange={e => onChange({ owner: e.target.value })}
-          value={owner}
-        />
-        <FormHelperText>The Owner that this repo will belong to</FormHelperText>
+        {allowedOwners?.length ? (
+          <Select
+            native
+            label="Owner"
+            onChange={s =>
+              onChange({ owner: String(Array.isArray(s) ? s[0] : s) })
+            }
+            disabled={allowedOwners.length === 1}
+            selected={owner}
+            items={ownerItems}
+          />
+        ) : (
+          <>
+            <InputLabel htmlFor="ownerInput">Project</InputLabel>
+            <Input
+              id="ownerInput"
+              onChange={e => onChange({ owner: e.target.value })}
+              value={owner}
+            />
+          </>
+        )}
+        <FormHelperText>
+          The project that this repo will belong to
```

Comment:
Let's keep the same label here

Reply:
I updated the form helper text.

Reply:
On second thought, I am not sure if you meant to change the label `Project` back to `Owner`. I'd prefer `Project` as it aligns with the concept in Azure DevOps. Backstage also uses the label `Project` instead of `Owner` for the Bitbucket repo picker.